espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/ElementValue.java

```diff
@@ -20,11 +24,27 @@ public ElementValue(boolean isReplacing) {
     @Override
     public Void handle(ElementValueParams params) throws AppiumException {
         String elementId = params.getElementId();
+        View view = Element.getViewById(elementId);
+
+        try {
```

Comment:
shall we probably move this block of code into a separate helper to avoid unnecessary duplication?

Reply:
@mykola-mokhnach Yeah I'm going to put in a PR for that.